.agents/skills/commit/SKILL.md

```
@@ -0,0 +1,49 @@
+---
+name: commit
+description: Create a git commit following the project's commit conventions. Use when the user asks to commit changes, or after completing a task that should be committed.
+allowed-tools: Bash(git:*), Glob, Read
+---
+
+# Git Commit
+
+Create a well-structured git commit. If necessary, create multiple commits to logically separate changes. Follow the project's commit message conventions, which are typically based on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
+Commit message should follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) conventions:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)